canonical-web-and-design/ubuntu.com
# Select a contract when clicking on a card

## Done

- Select a contract when clicking on a card.
- Add selected state styles.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10198.demos.haus/advantage?test_backend=true).
- You should be able to click between the two cards on the left.
- When a card has been click on it should show the [selected state](https://app.zeplin.io/project/60ec5fb4e07cc90fbbf2b318/screen/60edc152de9015ae8a35ccd2).
- Tab to the unselected card and press enter or spacebar. It should get selected.


## Issue / Card

Fixes: https://github.com/canonical-web-and-design/commercial-squad/issues/102

## Screenshots

<img width="380" alt="Screen Shot 2021-08-16 at 4 49 45 pm" src="https://user-images.githubusercontent.com/361637/129522689-d2158310-9e2d-4393-994a-d11c7b4ed2ce.png">
